components/common/heading/RenderHeading

```diff
@@ -0,0 +1,79 @@
+import React from "react";
```

Comment:
Add the file extension `.js`.

Add the license statement at the top. You can get it from [here](https://github.com/ballerina-platform/ballerina-dev-website/blob/ace4e0e7efcad1503bd5ad0462e1b0998cd8b34d/components/common/footer/Footer.js#L1). change the year to 2023.

Reply:
License Statement added in `RenderHeading` component in new PR.

Reply:
License Statement added in `RenderHeading` component in new PR.

Reply:
@JayPokale Add the file extension `.js` to the filename

Reply:
Sorry for this mistake. It's now corrected.